tests/integration/features/istio/evaluation/istio_installation_demo.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Installing Istio module with evaluation profile
```

Comment:
As far as I understand we are not using this test at the moment. I won't add tests that are not executed continuously, because this can lead to incorrect safety assumptions. I think if we want to keep this test, we should at least plan to add a separate Prow job for this.

Reply:
Created a PR with the prow job: https://github.com/kyma-project/test-infra/pull/7869
Prow job run: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_istio/174/barchw_test_of_prowjob_pull-istio-integration-evaluation/1668552806149132288